client/ayon_core/lib/path_templates.py

```diff
@@ -460,6 +460,30 @@ def validate_value_type(value):
                 return True
         return False
 
+    @staticmethod
+    def validate_key_is_matched(key):
+        """
+        Finds out how balanced an expression is.
+        With a string containing only brackets.
+
+        >>> is_matched('[]()()(((([])))')
+        False
+        >>> is_matched('[](){{{[]}}}')
+        True
+        """
+        opening = tuple('({[')
+        closing = tuple(')}]')
```

Comment:
Double quotes are preferred.
```suggestion
        opening = tuple("({[")
        closing = tuple(")}]")
```

Reply:
Noted !